apps/codeforafrica/src/lib/data/blockify/index.js

```diff
@@ -18,6 +18,9 @@ const propsifyBlockBySlug = {
 };
 
 async function blockify(blocks, api, context) {
+  if (!blocks) {
+    return null;
+  }
   const promises = blocks?.map(async (block) => {
```

Comment:
Doesn't `blocks?.` already handle cases where blocks is `null`/`undefined`?

Reply:
I had a case where blocks is undefined resulting in unserializable error.

Reply:
That's because we're returning `blocks` i.e. if `blocks` was `undefined`, we'll return `undefined`. If it was `null`, we'll return `null`.

Change it to `return blocks ?? null` if we don't want to return `undefined` under any circumstances.